templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -61,6 +61,17 @@ contents:
       fi
     }
 
+    #backup latest static pod resources for kube-apiserver
+    backup_latest_kube_static_resources() {
+      echo "Trying to backup latest static pod resources.."
+      LATEST_STATIC_POD_DIR=$(ls -td "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]* | head -1) || true
+      if [ -z "$LATEST_STATIC_POD_DIR" ]; then
+          echo "error finding static-pod-resources"
+          exit 1
+      fi
+      tar -czf $ASSET_DIR/backup/kube_static_resources.tar.gz $LATEST_STATIC_POD_DIR
```

Comment:
I think we should consider adding timestamp to directories so backups do not overwrite themselves by default

Reply:
Do we need to consider permissions/ownership here at all with `-p`

Reply:
Yeah. -p also retains the date-timestamps.